app/views/partials/_navbar.html.erb

```diff
@@ -0,0 +1,37 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light justify-content-between">
+    <div class='d-flex flex-row gap-3' style="align-items: baseline;">
+      <i class="fa-regular fa-bowl-food"></i>
+      <a class="navbar-brand logo" href="#">Tasty Trek</a>
+      <% if user_signed_in? %>
+        <p class='border border-success p-2 rounded'>Welcome, <%= current_user.name%></p>
+      <%end%>
+    </div>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <div class="navbar">
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item">
+        <%= link_to 'Home', root_path, class: 'nav-link'%>
+      </li>
+      <li class="nav-item">
+        <%= link_to 'Foods', food_index_path, class: 'nav-link'%>
+      </li>
+      <li class="nav-item">
+        <%= link_to 'My Recipes', recipes_path, class: "nav-link"%>
+      </li>
+      <li class="nav-item">
+        <%= link_to 'About Us', root_path, class: "nav-link"%>
+      </li>
+      <% if user_signed_in? %>
+      <li class="nav-item d-flex justify-center">
+        <%= link_to 'Sign out', destroy_user_session_path, method: :delete, class: "btn btn-outline-success pb-0 pt-2"%>
+      </li>
+      <% else %>
+        <%= link_to "Sign in", new_user_session_path, class: "btn btn-outline-success mx-2 pb-0 pt-2" %> 
+        <%= link_to "Sign up", new_user_registration_path, class: "btn btn-outline-success mx-2 pt-2" %>
+      <%end%>
+    </ul>
```

Comment:
- [ Required ] Please consider adding a page that shows a listing of all the `public` recipes, as part of the project requirements you are supposed to create a page [http://localhost:3000/public_recipes](http://localhost:3000/public_recipes) that shows all the public recipes as shown in the wireframe below.

## Project Requirements
![Screenshot from 2023-02-18 02-26-33](https://user-images.githubusercontent.com/16392046/219817495-b11598b2-55d6-4db8-ad4a-91ea0c09b89b.png)

![](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/public_recipe_index.png?raw=true)

Reply:
Thank you so much, I did all the required changes.